.github/CONTRIBUTING.md

```diff
@@ -1,28 +1,19 @@
-
+# Contributing Guide
 Hey Contributor! :smiley_cat:
 
-All contribution to PHPSA are very much encouraged, and we do our best to make it as welcoming and simple as possible.
+All contributions to PHPSA are very much encouraged, and we do our best to make it as welcoming and simple as possible.
 
 ## Coding Standards
 
 We require that all contributions meet at least the following guidelines:
 
 * Follow PSR-1 & PSR-2
 * Use camelCase for variables and methods/functions.
-* Don't use functions for casting like `intval`, `boolval` and etc, We are using `(int) $a`.
```

Comment:
I suggest to dont remove this
Because new developers dont known about all analyzers